File: LV1/K번째수.js
function solution(array, commands) {
  let answer = [];

  for (let i = 0; i < commands.length; i++) {
    let c = array
      .slice(commands[i][0] - 1, commands[i][1])
      .sort((a, b) => a - b); //2,3,5,6

    answer.push(c[commands[i][2] - 1]); //c 0,2 => 3 3번째 값이 인덱스 3이 아님
  }

  return answer;
}

File: LV1/K번째 수.java
class Solution {

  public int[] solution(int[] array, int[][] commands) {
    int[] answer = new int[commands.length];
    for (int i = 0; i < commands.length; i++) {
      int[] command = commands[i];
      int[] l = new int[command[1] - command[0] + 1];
      for (int j = 0; j < l.length; j++) {
        l[j] = array[command[0] + j - 1];
      }
      Arrays.sort(l);
      answer[i] = l[command[2] - 1];
    }
    return answer;
  }
}
